rs/tests/driver/src/driver/logger.rs

```diff
@@ -39,15 +39,23 @@ fn new_file_logger<P: AsRef<Path>>(p: P) -> Result<Logger> {
     Ok(slog::Logger::root(async_drain(file_drain), o!()))
 }
 
+pub fn new_discard_logger() -> Logger {
+    slog::Logger::root(async_drain(slog::Discard), o!())
+}
+
 fn multiplex_logger(l1: Logger, l2: Logger) -> Logger {
     slog::Logger::root(slog::Duplicate(l1, l2).fuse(), o!())
 }
 
 /// creates a slog::Logger that prints to standard out using an asynchronous drain
-pub fn new_stdout_logger() -> Logger {
-    let decorator = slog_term::TermDecorator::new().build();
-    let drain = SysTestLogFormatter::new(decorator).fuse();
-    slog::Logger::root(async_drain(drain), o!())
+pub fn new_stdout_logger(quiet: bool) -> Logger {
+    let decorator = slog_term::TermDecorator::new().force_color().build();
```

Comment:
I force color, because I wasn't able to get my terminal to register any other way, but maybe that is just me.